networkx/algorithms/broadcasting.py

```diff
@@ -30,34 +29,37 @@ def _get_max_broadcast_value(G, U, v, values):
 def _get_broadcast_centers(G, v, values, target):
     adj = sorted(G.neighbors(v), key=values.get, reverse=True)
     j = next(i for i, u in enumerate(adj, start=1) if values[u] + i == target)
-    return set([v] + adj[:j])
+    return {v} | set(adj[:j])
```

Comment:
Why change this?  My quick tests show it is slower. And its not any more clear.

Reply:
Thanks for benchmarking - I can confirm there's about a ~50% slowdown (which is frankly a lot more than I'd have expected.)
I changed it because I thought the "allocate list then convert to set" pattern would be slower if there are lots of duplicates that need to be ignored (which is true, about 30% slower according to my tests just now)... but failed to see that there wouldn't be any duplicates in this case. I've reverted the change!